src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue

```diff
@@ -122,22 +122,18 @@ export default {
   },
   computed: {
     getValidators() {
-      if (this.transaction.value.length > 1) {
+      if (this.transaction.withdrawValidators) {
         let validators = []
-        this.transaction.value.forEach(msg => {
+        JSON.parse(this.transaction.withdrawValidators).forEach(msg => {
```

Comment:
why does `withdrawValidators` not just hold all validator addresses?

Reply:
What do you mean? Why do you say it doesn't?

Reply:
`this.transaction.withdrawValidators` is returning a list of messages which is not necessary as the FE only needs the validator addresses. It should return just an array of validator addresses.

Reply:
Oh, I see what you mean... `withdrawValidators` PR has but been already merged and we are going to refactor all this in no time though...

So I think it is OK to pass this for this one time and merge this ^^

Reply:
ok